spiffe-helper.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: spiffe-helper
+  version: 0.9.0
+  epoch: 0
+  description: A helper utility for SPIFFE (Secure Production Identity Framework For Everyone) operations.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/spiffe/spiffe-helper.git
+      tag: v${{package.version}} 
+
+  - uses: go/build
+    with:
+      packages: "./cmd/spiffe-helper"
+      ldflags: "-s -w"
```

Comment:
```suggestion

```
we can omit this because we don't want `-s` because we don't strip symbols (for better sbom discovery) and `-w` comes by default from go/build pipeline.